packages/mui-material/src/StepLabel/StepLabel.d.ts

```diff
@@ -49,6 +68,11 @@ export interface StepLabelProps extends StandardProps<React.HTMLAttributes<HTMLD
      */
     label?: React.HTMLProps<HTMLSpanElement>;
   };
+  /**
+   * The components used for each slot inside.
+   * @default {}
+   */
+  slots?: {};
```

Comment:
This should come from `StepLabelSlotsAndSlotProps`, right?

Reply:
removed, since `slotProps` is also coming from `StepLabelSlotsAndSlotProps` i've removed that too. Checked `Alert` and `Avatar` both are following same pattern.